docs/geedocs/5.2.1/answer/2987876.html

```diff
@@ -1,66 +1,66 @@
 <!DOCTYPE html> <html> <head> <meta charset="UTF-8"/>
-<title>Legal notices</title>
```

Comment:
The changes to these 2 files should also be applied to the same files in the 5.2.2 directory

Reply:
yes, now that the pr of 5.2.2 doc is merged (https://github.com/google/earthenterprise/pull/748/), I'll make these changes there as well.

Reply:
done

Reply:
You didn't copy the indention changes into the file in 5.2.2. Was that intentional? It's really just a style choice, but if you want it to live on, it will have to be done :)

Reply:
I'm going to go ahead and approve this to minimize the extra work that will have to be done down the line. If you want the indention changes in 5.2.2 and subsequently 5.2.3, then open up a PR on the appropriate branch.